DnD-Montreal/session-tome
# Add Rating Report Generation

## Description
Closes #208 
Adds a set of actions that stream a CSV file to the users browser. The CSV file is a compilation of all ratings given to users. The report is only able to be generated for admins.

## How to Test
1) Ensure you have a freshly migrated and seeded database (`php artisan migrate:fresh && php artisan db:seed`)
2) Run the `RoleSeeder` to generate the appropriate roles `php artisan db:seed --class=RoleSeeder`
3) Use tinker to attach the admin role to a user `User::first()->roles()->save(Role::first())`
4) Login as that user on `/admin`
5) Goto the User Index
6) Click the Rating Report button
7) Check the CSV file thats downloaded, it should have a couple users and their compiled ratings in it.
